CMakeLists.txt

```diff
@@ -66,17 +66,6 @@ if(NOT CHFL_SYSTEM_LZMA)
     add_definitions("-DLZMA_API_STATIC")
 endif()
 
-if(CHFL_CODE_COVERAGE)
-    message(STATUS "Code coverage enabled")
-    if(NOT CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
Is this check still required?

Reply:
The idea is that the users can set `C_FLAGS/CXX_FLAGS` if they want code coverage. If so, the flag will be checked by the compiler directly